server/api.js

```diff
@@ -1,25 +1,23 @@
-'use strict'
+'use strict';
 
-const db = require('APP/db')
-const api = module.exports = require('express').Router()
+const db = require('APP/db');
+const api = module.exports = require('express').Router();
 
 api
   .get('/heartbeat', (req, res) => res.send({ok: true,}))
   .use('/auth', require('./auth/auth'))
   .use('/users', require('./routes/users'))
-
-  .use('/addresses', require('./routes/addresses'))
   .use('/carts', require('./routes/carts'))
```

Comment:
We should also delete /carts, /payments (carts and payments models were deleted)